### PR TITLE
Client init message

### PR DESCRIPTION
**Pull request by ax487**
Hopefully fixing https://github.com/google/xi-editor/issues/353

### CODE REVIEW

**Comment by ax487** [comment]
I signed it!

File: rust/core-lib/src/rpc.rs [hunk 1]
```diff
@@ -175,9 +176,11 @@ impl<'a> CoreCommand<'a> {
...
                         } else { Err(MalformedCoreParams(method.to_string(), params.clone())) }
                 }),
-                "plugin" => serde_json::from_value::<PluginCommand>(params.clone())
-                    .map(|cmd| Plugin { plugin_command: cmd })
-                    .map_err(|_| MalformedPluginParams(method.to_string(), params.clone())),
+            "plugin" => serde_json::from_value::<PluginCommand>(params.clone())
+                .map(|cmd| Plugin { plugin_command: cmd })
+                .map_err(|_| MalformedPluginParams(method.to_string(), params.clone())),
+
+            "client_started" => Ok(ClientStarted{}),
```
Comment (cmyr) [inline rust/core-lib/src/rpc.rs]: I'm mildly surprised to see that this compiles (including the braces `{}` in a a non-struct enum member). I'm tempted to say remove them, for clarity, but it's a bit of a nit.

**Comment by ax487** [comment]
@cmyr : Done.

**Comment by cmyr** [comment]
@ax487 great, will get this in when we get the ✅ from CI & @raphlinus. 😄

**Comment by raphlinus** [comment]
Thanks!

**Pull request merged**